config/environments/development.rb

```diff
@@ -75,13 +75,14 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: config.app_domain }
   config.action_mailer.smtp_settings = {
-    address: "smtp.gmail.com",
```

Comment:
I'm guessing this was due to it not being a deployed live site, but this allows it to  function locally? Nice job working that out!